src/avt/Filters/avtCondenseDatasetFilter.C

```diff
@@ -224,93 +229,59 @@ avtCondenseDatasetFilter::ExecuteData(avtDataRepresentation *in_dr)
             }
         }
     }
- 
+
     //
     // Decide whether or not we need to determine the relevant points.
```

Comment:
we should probably remove this comment

Reply:
Well, we still determine whether or not to use relevant points filters: there has to be cells, and the mesh must be Unstructured Grid or PolyData.

Reply:
Ahh, that's right, thanks for clarifying.